### PR TITLE
Versions now in reports

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 339484efb8875c5d782fd2cc755f17bd186113f8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 263c85e2fea43ec42284ce4584984fdfe26f09df:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 22bccf4e506853d46b46bd941b36eacf134031f5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2997c172401ab4d0bbe79d41aea093cc3703906e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: data/templates/test_report.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
   <div class="row">
     <div class="col-md-4">
       <h1>Test Report</h1>
+      <h2>Version: {{version}}</h2>
```
Comment (seandst) [inline data/templates/test_report.html]: Was about to merge, but just noticed this. Now that version is optional, this needs to be `{% if version %}<h2>Version: {{version}}</h2>{% endif %}`, then :+1:

**Comment by dajoRH** [comment]
Lint report for commit 2e2c9700accac68355a34552623561bdd1aef71f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**